recipe/meta.yaml

```
@@ -21,24 +21,24 @@ source:
   fn: wheel.whl
 
   url: https://pypi.io/packages/cp36/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-cp36-none-{{ linux_whl }}.whl  # [linux and py==36]
-  sha256: 0f83f307f3d5864a05dca7a70a667c4c7e7de3cdd5c116b1ec78fc18b6da57cc  # [linux and py==36]
+  sha256: 05858e34e542ed9fd54b066efa9a53ac6297eb5d6c1ee83f67808fc4d17a12bc  # [linux and py==36]
```

from https://pypi.org/project/jaxlib/#copy-hash-modal-2e628ed5-240a-4c4a-a5da-04d66119f6c8

```suggestion
  sha256: e41830b8b02a44da36b73bf8dc268c3acc9d85194bb96cbed07f1ab24ce6be53  # [linux and py==36]
```